src/projectDB/projectdatabase.cpp

```diff
@@ -113,7 +113,7 @@ void ProjectDatabase::saveSamples(const vector<mzSample*>& samples)
         samplesQuery->bind(":color_blue", s->color[2]);
         samplesQuery->bind(":color_alpha", s->color[3]);
 
-        samplesQuery->bind(":normal_const", s->getNormalizationConstant());
```

Comment:
@saifulbkhan Is this intentional?

Reply:
Yes. Unfortunately, our schema was initialized with `norml_const` as the column for storing the scaling factor, but the save method was using `normal_const` (which should have been the correct one). Making this change in the schema sounds like too much work for a spelling mistake, but is possible.